### DISCUSSION TITLE
Nuxt deployment in AWS Amplify

### DISCUSSION THREAD

**Discussion by ivsaneesh** [nuxt/nuxt#19728] [Questions]
Nuxt 3 is what i'm using. Could someone point me in the right direction? 
I'm confused at what I should set the source directory path and distribution directory path to when trying to deploy to AWS Amplify.

**Comment by imlautaro** [comment]
Not sure if there's a way to deploy Nuxt SSR apps to AWS Amplify, but if you're using SSG (nuxi generate) your dist path should be `dist`, and your source directory should be the root of your project or if you have a srcDir configured, that path. If I'm not wrong the src dir you configure in Amplify is just to put there TypeScript interfaces and stuff like that.

**Comment by imlautaro** [comment]
In case you wanna deploy a Nuxt SSR app, please refer to Nitro's deployment guide for AWS Lambda: https://nitro.unjs.io/deploy/providers/aws
